src/Service/VendorService/BogPortalen/BogPortalenVendorService.php

```diff
@@ -224,4 +227,30 @@ private function getIsbnNumbers(array &$filePaths): array
         // https://stackoverflow.com/questions/8321620/array-unique-vs-array-flip
         return array_flip(array_flip($isbnList));
     }
+
+    /**
+     * {@inheritDoc}
+     */
+    public function getUnverifiedVendorImageItem(string $identifier, string $type): UnverifiedVendorImageItem
+    {
+        if (!$this->supportsIdentifierType($type)) {
+            throw new UnsupportedIdentifierTypeException('Unsupported single identifier type: '.$type);
+        }
+
+        $item = new UnverifiedVendorImageItem();
+        $item->setIdentifier($identifier);
+        $item->setIdentifierType($type);
+        $item->setVendor($this->vendorCoreService->getVendor(self::VENDOR_ID));
```

Comment:
Here I prefer to have it in an $vendor variable for readability.

Reply:
Fixed (also moved public functions up before private functions)